index.html

```diff
@@ -1,179 +1,160 @@
+          <article class="article">
+            <h3 class="article__title">Why do refugees need bikes?</h3>
+            <p class="article__summary">
+              Many refugees are placed in housing in areas where there are
+              few jobs. Bikes provide low-cost transportation so that they
+              can get to work, manage child care and do their shopping.
+            </p>
+            <p class="article__read-more">
+              <a href="#">
+                <span class="arrow-right"></span>
+                Learn more
+              </a>
+            </p>
+          </article>
+
+          <article class="article">
+            <h3 class="article__title">How can I help?</h3>
+            <p class="article__summary">
+              We need lots of bikes and bike accessories! If you have an
+              old bike you don't use, bring it to one of our dropoff
+              events. Or come attend one of our fundraisers.
+            </p>
+            <p class="article__read-more">
+              <a href="#">
+                <span class="arrow-right"></span>
+                Learn more
+              </a>
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
-          </div>
-
-          <div>
-            <h2 class="heading-underline">Learn more</h2>
-
-            <div>
-              <div class="article">
-                <h3 class="article__title">Why do refugees need bikes?</h3>
-                <p class="article__summary">
-                  Many refugees are placed in housing in areas where there are
-                  few jobs. Bikes provide low-cost transportation so that they
-                  can get to work, manage child care and do their shopping.
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
-
-              <div class="article">
-                <h3 class="article__title">How can I help?</h3>
-                <p class="article__summary">
-                  We need lots of bikes and bike accessories! If you have an
-                  old bike you don't use, bring it to one of our dropoff
-                  events. Or come attend one of our fundraisers.
-                </p>
-                <p class="article__read-more">
-                  <a href="#">
-                    <span class="arrow-right"></span>
-                    Learn more
-                  </a>
-                </p>
-              </div>
-            </div>
-          </div>
+          </article>
+        </section>
+      </section>
+    </main>
+
+    <aside class="sidebar">
+      <h2 class="heading-underline">Upcoming events</h2>
+
+      <article class="article">
+        <img class="article__thumbnail" src="images/spring-fundraisers_thumbnail.jpg" alt="spring fundraisers" />
```

Comment:
Good correction of image path and use of alt text

Reply:
Thank you very much